test/exchangeIssuance/exchangeIssuance.spec.ts

```diff
@@ -258,16 +257,27 @@ describe("ExchangeIssuance", async () => {
     let wbtc: StandardTokenMock;
     let dai: StandardTokenMock;
     let usdc: StandardTokenMock;
+    let illiquidToken: StandardTokenMock;
+    let setTokenIlliquid: SetToken;
 
     beforeEach(async () => {
       let uniswapSetup: UniswapFixture;
       let sushiswapSetup: UniswapFixture;
 
-      // TODO: should we instead port SystemFixtrue and use tokens from it ?
       weth = setV2Setup.weth;
       wbtc = setV2Setup.wbtc;
       dai = setV2Setup.dai;
-      usdc = await deployer.setV2.deployTokenMock(user.address, 1000000 * 10 ** 6, 6, "USD Coin", "USDC");
+      usdc = setV2Setup.usdc;
+      illiquidToken = await deployer.setV2.deployTokenMock(owner.address, ether(1000000), 18, "illiquid token", "RUGGED");
```

Comment:
A set token can be illiquid when 
1. It is not present on either of the exchanges (you have accounted for this scenario)
2. We want a certain amount of it, but the amount present on both the exchanges individually, is lesser than our required amount.
Please include tests for the second case as well.